### DISCUSSION TITLE
Do I need to rewrite balanced data after Minio peer-to-peer expansion

### DISCUSSION THREAD

**Discussion by smallemployees** [minio/minio#20462] [Q&A]
## NOTE
If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.

## Expected Behavior

How to distribute the load to all nodes
## Current Behavior

Uneven pressure on the requested IO
## Steps to Reproduce (for bugs)

1.
2.
3.
4.

## Your Environment

* Version used (`minio --version`): RELEASE.2023-03-22T06-36-24Z
* Server setup and configuration:
* Operating System and version (`uname -a`):

**Comment by mrf-cyber** [comment]
Minio distribute more IO on the new disk/ empty nodes. If you wanna get all nodes working with the same preasure you need check:

[Minio Reblanace](https://min.io/docs/minio/linux/reference/minio-mc-admin/mc-admin-rebalance.html)

Once finish all new IOs will be distributed across all pools. Minio dont autorebalance data over new nodes.
